core/src/main/java/com/nisovin/magicspells/castmodifiers/conditions/UsingCondition.java

```diff
@@ -1,47 +1,50 @@
 package com.nisovin.magicspells.castmodifiers.conditions;
 
-import com.nisovin.magicspells.castmodifiers.Condition;
-import com.nisovin.magicspells.util.Name;
-import com.nisovin.magicspells.util.magicitems.MagicItemData;
-import com.nisovin.magicspells.util.magicitems.MagicItems;
+import org.jetbrains.annotations.NotNull;
+
 import org.bukkit.Location;
 import org.bukkit.entity.LivingEntity;
 import org.bukkit.inventory.ItemStack;
-import org.jetbrains.annotations.NotNull;
+
+import com.nisovin.magicspells.util.Name;
+import com.nisovin.magicspells.castmodifiers.Condition;
+import com.nisovin.magicspells.util.magicitems.MagicItems;
+import com.nisovin.magicspells.util.magicitems.MagicItemData;
 
 @Name("using")
 public class UsingCondition extends Condition {
 
-	private MagicItemData itemData = null;
-	
+	private MagicItemData itemData;
+
 	@Override
 	public boolean initialize(@NotNull String var) {
 		itemData = MagicItems.getMagicItemDataFromString(var);
-		return itemData != null;
+		return true;
```

Comment:
This should be `itemData != null || var.isEmpty()`. An invalid magic item should cause an error.

Reply:
Actually, probably should do:
```java
if (var.isEmpty()) return true;
itemData = MagicItems.getMagicItemDataFromString(var);
return itemData != null;
```